inc/classes/class-settings.php

```diff
@@ -112,5 +122,28 @@ public function cse_id_field_cb() {
 		<input type="text" name="gcs_cse_id" value="<?php echo ! empty( $setting ) ? esc_attr( $setting ) : ''; ?>">
 		<?php
 	}
+
+	/**
+	 * Search Type field markup.
+	 *
+	 * @return void
+	 */
+	public function cse_search_type_field_cb() {
+		// Get the value of the setting we've registered with register_setting().
+		$setting = get_option( 'gcs_search_type' );
+		// Add slider toggle switch.
+		?>
+		<div class="switch-wrapper">
+			<span><?php esc_html_e( 'Custom Site Restricted Search API', 'search-with-google' ); ?></span>
+
+			<label class="switch">
+				<input id="search-type" type="checkbox" name="gcs_search_type" value="1" <?php echo ! empty( $setting ) ? 'checked' : ''; ?>>
```

Comment:
@SH4LIN Can we use [checked()](https://developer.wordpress.org/reference/functions/checked/) here.